lib/entryList.ts

```diff
@@ -0,0 +1,50 @@
+import { Channel } from './channel'
+import {
+  EntryListAPI,
+  OnEntryListUpdatedHandler,
+  OnEntryListUpdatedHandlerProps,
+  OnEntryListUpdatedHandlerReturn,
+} from './types'
+
+const isFunction = (f: any) => typeof f === 'function'
+
+const runHandler = async (
+  handler: OnEntryListUpdatedHandler | null,
+  handlerArg: OnEntryListUpdatedHandlerProps
+) => {
+  try {
+    // Handler was not registered. Registering a handler is required.
+    if (!handler) {
+      throw new Error('Registering an onEntryListUpdated handler is required')
```

Comment:
You are right, now it throws the error on each app initialization, because the message gets called before we attach the handler. I think we can remove it, right?

Reply:
Yes, let's remove the `throw` and simply return here